### PR TITLE
Add meta charset utf-8 to all pages

### PR DESCRIPTION
**Pull request by lencioni**
We are currently seeing a weird encoding issue on the uploaded diffs
page. I'm not sure if this will resolve that, but it is the right thing
to do and it might help.

Addresses #121

### CODE REVIEW

File: lib/happo/views/diffs.erb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <html>
   <head>
     <title><%= Happo::Utils.page_title(diff_images, new_images) %></title>
+    <meta charset="utf-8" />
```
Comment (trotzig) [inline lib/happo/views/diffs.erb]: I think it would be better to put the meta tag above the title.

**Comment by trotzig** [comment]
Did you try dynamically adding this to your broken diff page? Maybe that won't have any effect though, I don't really understand how you can switch encoding mid-document...

File: lib/happo/views/diffs.erb (hunk 1 above)
Comment (lencioni) [inline lib/happo/views/diffs.erb]: Good call. Done!

**Comment by trotzig** [comment]
👍

**Comment by lencioni** [comment]
I don't think that would have any affect. But I'll give it a shot.

**Comment by lencioni** [comment]
Dynamically adding it did nothing.

**Pull request merged**